pkg/ipam/ipam_linux.go

```
@@ -115,7 +115,8 @@ func ConfigureIface(ifName string, res *current.Result) error {
 
 	if v6gw != nil {
 		err = ip.SettleAddresses(ifName, dadSettleTimeout)
```

if the kernel leaves the address in tentative state when there is no carrier, why don't you add this check inside SettleAddresses. Check if the oper is up and the interface is tentative, then ignore error